guide/migration/20-to-30.md

```diff
@@ -98,6 +98,20 @@ services.AddOpenIddict()
     });
 ```
 
+## Enable request caching
```

Comment:
Should the section title be updated to something that doesn't imply it's a mandatory feature? Maybe "If applicable, use the new request caching activation methods"?

Reply:
Good idea. Changed description. Is it better now?